src/Uno.UI/UI/Xaml/Controls/ComboBox/ComboBox.cs

```diff
@@ -280,25 +272,17 @@ protected override void OnPointerReleased(PointerRoutedEventArgs e)
 						popupChild.MaxHeight = Math.Min(MaxDropDownHeight, visibleBounds.Height * 0.6);
 
 						var popupRect = popup.GetAbsoluteBoundsRect();
-						var backgroundRect = background.GetAbsoluteBoundsRect();
-
-						// Because Popup.Child is not part of the visual tree until Popup.IsOpen,
-						// some descendent Controls may never have loaded and materialized their templates.
-						// We force the materialization of all templates to ensure that Measure works properly.
-						foreach (var control in popupChild.EnumerateAllChildren().OfType<Control>())
```

Comment:
It's safe to remove this? Did you test on Android?

Reply:
It's was a duplicate, it's already done few lines before : https://github.com/nventive/Uno/pull/1043/files/be24bfe96ccdc57acba300731f8d06502bc08265#diff-4c39c115cd542f80d1c8a8990486f82fR239